Mariia_Andanchenko/Homework12/task2.py

```diff
@@ -0,0 +1,76 @@
+class Worker:
+    """
+    The Worker class contains the following information about the employee:
+    - name
+    - age
+    - work experience
+    - salary
+    This class allows:
+    - see the name and length of service of the employee
+    - get a salary
+    - do some work
+    - take a vacation
+    - get bonuses
+    - get a salary increase
+    - celebrate a birthday
+    """
+
+    def __init__(self, name: str, age: int, length_work: float, salary: float):
+        self.__name = name
+        self.__age = age
+        self.__length_work = length_work
+        self.__salary = salary
+
+    @property
+    def name(self):
+        return self.__name
+
+    @property
+    def length_work(self):
+        return self.__length_work
+
+    def get_salary(self):
+        """
+        This function allows the employee to receive a salary
+        """
+        print(f'Співробітник {self.__name} отримав зарплатню у розмірі {self.__salary}грн')
+
+    def working(self):
+        """
+        This function allows do some work
+        """
+        print(f'Співробітник {self.__name} виконує свої обов\'язки')
+
+    def take_vacation(self, days: int):
+        """
+        This function allows the employee to take vacation. The number of vacation days is passed to the function
+        """
+        print(f'Співробітник {self.__name} взяв відпустку на {days} днів')
+
+    def get_bonus(self, bonus: float):
```

Comment:
in your implementation, this function can be private. So nobody can pay lots of bonuses without control